src/pages/shopping-cart/wnft.vue

```diff
@@ -207,6 +207,7 @@ export default {
   mounted() {
     this.updateLIKEPrice();
     logPurchaseFlowEvent(this, 'add_to_cart', this.purchaseEventParams);
```

Comment:
Should we keep triggering `add_to_cart` when entering this page?

Reply:
In wnft I think the user would always go into cart page when clicking `purchase all`?
But wnft flow is not very important now i guess